platform-server-rest/src/main/java/uno/cod/platform/server/rest/RestUrls.java

```diff
@@ -22,6 +22,9 @@
 
 
     public static final String USERS = "/users";
+    public static final String USERS_USERNAME = "/users/{username}";
+    public static final String USERS_ID_TEAMS = "/users/{id}/teams";
```

Comment:
If we opt for `/users/{username}` only, i.e. ditch `/users/{id}`, we should also switch from `/users/{id}/teams` to `/users/{username}/teams` IMHO.